SU-SWS/stanford_components
# Earth 16 fw banner

# READY FOR REVIEW
# Summary
- Changed the pattern labels, changed the way the `items` field renders. This will help when using field groups or embeded paragraphs.

# Needed By (Date)
- End of sprint

# Urgency
- Low urgency

# Steps to Test

1. checkout branch, `drush cr`
2. verify the /patterns page still looks accurate
3. run tests on https://github.com/SU-SWS/stanford_paragraph_types/pull/33
4. verify the pattern looks good in the paragraph item.

# See Also
- https://github.com/SU-SWS/stanford_paragraph_types/pull/33
